packages/common/src/sites/HubSites.ts

```diff
@@ -309,35 +309,38 @@ export async function updateSite(
   site: IHubSite,
   requestOptions: IHubRequestOptions
 ): Promise<IHubSite> {
+  // verify that the slug is unique, excluding the current site
+  site.slug = await getUniqueSlug(
```

Comment:
Dave told me that we should use the same slug verification logic as projects, so I introduced it here

Reply:
@dbouwman, @mjuniper wasn't aware that sites had / needed slugs. Just to confirm before we merge, can we expect _every_ site to have a slug in `item.properties.slug`, or just enterprise sites?

Reply:
All "Entity" based items should create a slug, and store it in `item.properties.slug` and a corresponding typekeyword - all done using the same set of helper functions. 

Whether or not anything uses that is a separate question that comes down to how we want urls to work. 

And while it's true that Site's on AGO are unlikely to ever use this property, I'm more inclined to have consistent code for Enterprise and AGO than to have yet-more permutations. :)